x-pack/plugins/security_solution/server/plugin.ts

```diff
@@ -129,6 +135,7 @@ export class Plugin implements ISecuritySolutionPlugin {
   private readonly endpointAppContextService = new EndpointAppContextService();
   private readonly telemetryReceiver: ITelemetryReceiver;
   private readonly telemetryEventsSender: ITelemetryEventsSender;
+  private readonly telemetryEventsSenderV2: IAsyncTelemetryEventsSender;
```

Comment:
Similar to @stephmilovic 's previous comment: 

```suggestion
  private readonly asyncTelemetryEventsSender: IAsyncTelemetryEventsSender;
```

Reply:
Good catch! Thanks